live-build/misc/ansible-roles/appliance-build.minimal-common/tasks/main.yml

```diff
@@ -233,3 +233,42 @@
   with_items:
     - { key: "net.ipv4.conf.all.promote_secondaries", value: "1" }
     - { key: "net.ipv4.conf.default.promote_secondaries", value: "1" }
+
+#
+# Configure command audit logging
+#
+# We want to record all commands executed on the appliance. Opt out for
+# setsid since all ExecuteUtils.execute wrap each call with setsid.
+#
+# Note we can't use the "service" module, as that fails when running
+# inside the chroot environment, so we enable auditd using "command".
+#
+- apt:
+    name: auditd
```

Comment:
This isn't _wrong_, but I'd suggest we install this by adding the package to the file: `live-build/base/config/package-lists/minimal.list.chroot` .. just so it's (imo) more consistent with how we install other packages for the "minimal" variant. What do you think?

Reply:
@prakashsurya  Seems logical.  I moved the auditd package install location as suggested above.